src/main/webapp/WEB-INF/views/admin/license/list.jsp

```diff
@@ -102,6 +102,9 @@
 		<!---->
 		<div class="btnLayout">
 			<span class="right">
+				<c:if test="${ct:isAdmin() and project.dropYn ne 'Y'}">
```

Comment:
There is no need to check the project status. Make the button visible only if user is admin.

Reply:
thank you for good review